android/assets/jsons/translations/template.properties

```diff
@@ -1314,7 +1314,7 @@ ConditionalsPlacement =
 # In what order should these conditionals between <> be translated?
 # Note that this example currently doesn't make sense yet, as those conditionals do not exist, but they will in the future.
 
-<if this city has at least [amount] specialists> <when at war> <when not at war> = 
+<if this city has at least [amount] specialists> <when at war> <when not at war> <while the empire is happy> = 
```

Comment:
Once again calling attention to the fact that when we rebuild translations, all existing order will be lost.
Translators will need to do it all user again for every new conditional.
I really think we should remove the sorting entirely, at least for now

Reply:
I have remarked right above the translation that this is the case, so translators can choose for themselves whether or not to add these translations. I agree that that is imperfect in the short term, but in the end this will allow for the best translations, and for then this way the logic is already in place.

Reply:
I don't know whether it is actually useful in general, but I can imagine for some languages it alleviates at least some of the weirdness the translation would otherwise have. For now I'll comment out things related to this, but unless a better solution is found I feel like this should be enabled again once we've added all the conditionals we plan on adding.